modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/core/utils/merge-component-variants.es.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import * as EditorVariant from '../components/PageRenderer/EditorVariant.es';
+
+export const mergeComponentVariants = (
```

Comment:
Why not `assembleComponentVariants`?

Reply:
I think `merge` is more like the idea of ​​mixing the variants because it’s a commonly used term.  But thinking about it now I think it would be better just with `mergeVariants`.